### ISSUE TITLE
Branching Order-of-Operations

### ISSUE THREAD

**Issue by mirjak** [ietf-tapswg/api-drafts#536]
In section 4.1.4 in the impl draft, we have:

"The order of operations for branching, where lower numbers are acted upon first, should be:

1. Alternate Paths
2. Protocol Options
3. Derived Endpoints"

However, I would say there is not really a strict ordering required. The text actually indicates that selection of the protocol is rather independent while there is usually a dependency between the interface and derived endpoint. I think this is related to my other issue about the use of the term path, because a path consists of both the local and remote endpoint and therefore there is of course a dependency. 

So in summary I'm not sure how useful I find this whole section as currently written...

**Comment by tfpauly** [comment]
Any implementation should have a consistent order of operations. This is a recommended example. Reference the arch text that says it needs to work the same way each time.

**Comment by tfpauly** [comment]
Reference ICE/HE
